packages/menu/src/MenuItem.tsx

```diff
@@ -205,19 +210,17 @@ const MenuItem = React.forwardRef(
     }: MenuItemProps,
     forwardedref,
   ) => {
-    const Root = href ? 'a' : 'button';
-
-    const anchorProps = href && {
+    const anchorProps = rest.href && {
       target: '_blank',
```

Comment:
While we're in here, I'm not sure it's right to indiscriminately set `target='_blank'`. We may want to make the default either open in the same tab, or by default parse the URL to see if this should be set.

I'm fine with leaving it for now if we just wanna make a ticket, but I wanted to note this while it's fresh.

Reply:
Outside the scope of this PR but we should have this discussion and decide what we want the default behavior here to be.

Reply:
Unsolicited opinion as a user: I would expect the default target to be set to `_self` with `rel` omitted, since for the majority of navigation I write, I stay within our application. 

From my usage, the most major exception would be usage in `mongo-menu`. That component is shared, though, so as a user I'd just expect `mongo-menu` to deal with that on my behalf.

Reply:
Think you're right about that @seanmhanson: https://jira.mongodb.org/browse/PD-459